### PR TITLE
Add remote entries support to vscode recent project extention

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

This PR add support for remote entries in the recent project extension.

## Type of change

- Bug fix / improvement for somebody else's extension. Extension author: @thomaspaulmann 

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by mattisssa** [comment]
Hey @thomaspaulmann! Could you please take a look at the PR as well? 
it's working well for me, I just don't have any remote entries 🤷

**Comment by pomdtr** [comment]
Here is a doc to get started with the feature: https://code.visualstudio.com/docs/remote/containers-tutorial

File: extensions/visual-studio-code-recent-projects/src/index.tsx [hunk 1]
```diff
@@ -27,28 +29,57 @@ export default function Command() {
   const folders = new Array<ReactNode>();
   const files = new Array<ReactNode>();
   const workspaces = new Array<ReactNode>();
+  const remoteEntries = new Array<ReactNode>();
 
   const recentEntries = getRecentEntries();
   recentEntries.forEach((entry) => {
-    if (isFolderEntry(entry) && existsSync(new URL(entry.folderUri))) {
-      folders.push(<ProjectListItem key={entry.folderUri} uri={entry.folderUri} />);
+    if (isFolderEntry(entry) && isRemoteEntry(entry)) {
```
Comment (thomaspaulmann) [inline extensions/visual-studio-code-recent-projects/src/index.tsx:36]: NIT: Maybe move the `isFolderEntry` check into the `isRemoteEntry` check?

**Review by thomaspaulmann — APPROVED** [review]
Nice. This looks good thanks for adding. Could you run `ray lint` locally? I think the code isn't formatted properly. Left also a small comment. Feel free to address it before I merge the PR.

**Comment by pomdtr** [comment]
Thanks for the review
Ray lint did not return any error:

```
🔎 Linting
✅ No linter errors
```

**Pull request merged**